index.html

```diff
@@ -4,64 +4,68 @@
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
+     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <script  src="js/index.js"></script>
   </head>
 
   <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-     <nav id ="navbar" >
-       <header>this documentation is a sum of all of mankind knowledge</header>
```

Comment:
-You can replace header with h2 tag (i.e `<h2>this documentation is a sum of all... </h2>`)
- And wrap nav tag with header tag (i.e. `<header><nav id="navbar"></nav></header>`)
- This will help you apply best practice using semantic html tags